runtime-light/tl/tl-functions.h

```diff
@@ -0,0 +1,29 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <cstdint>
+
+#include "runtime-core/runtime-core.h"
+#include "runtime-light/tl/tl-core.h"
+
+namespace tl {
+
+constexpr uint32_t K2_INVOKE_HTTP_MAGIC = 0xd909efe8;
+constexpr uint32_t K2_INVOKE_JOB_WORKER_MAGIC = 0x437d7312;
+
+struct K2InvokeJobWorker final {
```

Comment:
Maybe rename it to make for clearer that it is request?

Reply:
It matches the name given this function in K2 platform.